test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Linux/AcceptanceTest.cs

```diff
@@ -223,7 +223,14 @@ public Task ResourceUtilizationTracker_And_Metrics_Report_Same_Values_With_Cgrou
         listener.InstrumentPublished = (Instrument instrument, MeterListener meterListener)
             => OnInstrumentPublished(instrument, meterListener, meterScope);
         listener.SetMeasurementEventCallback<double>((m, f, tags, _)
```

Comment:
Formatting: TBH I'm most used to arrow being in the same line where parameters are. I think it's more prevalent in this repo as well:
```suggestion
listener.SetMeasurementEventCallback<double>((m, f, tags, _) => ...
```
or 
```suggestion
listener.SetMeasurementEventCallback<double>((m, f, tags, _) => 
{
}
```